weaviate/helm/weaviate/values.yaml.tpl

```diff
@@ -1,5 +1,13 @@
 {{ $isGcp := or (eq .Provider "google") (eq .Provider "gcp") }}
 
+{{ $key := dedupe . "weaviate.weaviate.admin.key" (randAlphaNum 20) }}
```

Comment:
so this value needs to be set here for dedupe to work properly

Reply:
I think dedupe will work because we wrap the chart twice, cf postgres.password dedupe used on other apps. What do you suggest?

Reply:
Ok oh I understand, my bad